clients/src/test/java/org/apache/kafka/common/security/authenticator/LoginManagerTest.java

```diff
@@ -153,6 +163,32 @@ public void testLoginManagerWithDifferentConfigs() throws Exception {
         verifyLoginManagerRelease(staticLogin2, 2, staticContext, configs2);
     }
 
+    @Test
+    public void testLoginException() throws Exception {
+        Map<String, Object> config = new HashMap<>();
+        config.put("sasl.jaas.config", dynamicPlainContext);
```

Comment:
I think we could use `SaslConfigs.SASL_JAAS_CONFIG` here instead of this plain string literal.

Reply:
Good point, thanks a lot for the pointer.